src/core/battle/state/apply.rs

```diff
@@ -339,9 +339,13 @@ fn apply_effect_wound(state: &mut State, id: Id, effect: &effect::Wound) {
 fn apply_effect_knockback(state: &mut State, id: Id, effect: &effect::Knockback) {
     assert!(state.map().is_inboard(effect.from));
     assert!(state.map().is_inboard(effect.to));
+
```

Comment:
Could you please remove this empty line? I have a "soft code style" (it's not written anywhere yet) that empty lines inside functions should be avoided.